.github/workflows/docker.yml

```diff
@@ -6,8 +6,8 @@ on:
       - rc*/*
   push:
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v20.15.10-rc5, v20.15.10-rc12+build1
```

Comment:
Is It possible using `+` as tag in golang. 
Because I'm not sure it is no problem when other service import the tag with `+`.

Reply:
I don't know that, but this pattern matches with `v20.15.10-rc12+build1`. If it cannot be used as version, this pattern should be fixed.

Reply:
You can use `+` in tag. This can be confirmed with https://github.com/loloicci/gomodexample/tags